aws-rotate-iam-keys.template.rb

```diff
@@ -21,11 +21,15 @@ class AwsRotateIamKeys < Formula
   devel do
     Dir.chdir(File.expand_path(File.join(File.dirname(__FILE__), '../'))) do
       url %x{git config --local --get remote.origin.url | tr -d '\n'}, using: :git, branch: "develop"
-      version %x{git describe develop --always | tr -d '\n'}
+      version "head"
     end
   end
```

Comment:
should this devel block just be removed because of issue #43?

Reply:
Yep, it definitely should be removed, if the code is ever merged. I'm not sure the code in this PR is destined for a merge anymore though, at least as it stands. The PR seems to have turned more into a discussion about how to simplify the build process and to make contributing easier.

How to deal with the deprecated devel blocks is a tricky one though, there are forks of this project using the devel software spec to allow simple installation of the latest development code from the fork for real world testing before pull requests are submitted back to the upstream project. That's both my fault, and my problem, I'll need to figure something out pronto, because the deprecation warning is a pain in the ass.